nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -619,21 +619,22 @@ define([
                             // Update the execMessage panel with details of the active jobs
                             controlPanel.setExecMessage(
                                 Jobs.createCombinedJobState(
-                                    jobManagerContext.model.getItem('exec.jobs.byStatus')
+                                    jobManagerContext.model.getItem('exec.jobs')
```

Comment:
Not sure if this was causing https://kbase-jira.atlassian.net/browse/DATAUP-484, but it is definitely stopping the job status line from being rendered!

Reply:
😖